frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -46,6 +47,13 @@ import { checkArbiterCluster, checkFlexibleScaling } from '../../../utils/common
 const queries = (() => Object.values(CAPACITY_INFO_QUERIES))();
 const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
 const getProvisionedCapacity = (value: number) => (value % 1 ? (value * 3).toFixed(2) : value * 3);
+const getSelectedKey = (
+  storageClass: StorageClassResourceKind,
+  deviceSets: DeviceSet[],
+): string => {
+  if (storageClass) return getName(storageClass);
+  return deviceSets?.[0]?.dataPVCTemplate?.spec?.storageClassName;
```

Comment:
If there are more than one device sets, then we should show the last device set's storage class.